app/assets/stylesheets/application.css.scss

```diff
@@ -35,14 +35,32 @@
   height: 100vh;
   position: relative;
 }
+ // h1{
+ //   font-family: 'Poiret One', cursive;
+ // }
 
+.trophy-case-container {
+  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 100vh;
+  position: relative;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.